crates/pallets/starknet/src/tests/mock/genesis.json

```diff
@@ -150,6 +150,10 @@
       "0x00000000000000000000000000000000000000000000000000000000000000AA",
       "0x06232eeb9ecb5de85fc927599f144913bfee6ac413f2482668c9f03ce4d07922"
     ],
+    [
+      "0x00000000000000000000000000000000000000000000000000000000000000BB",
+      "0x06232eeb9ecb5de85fc927599f144913bfee6ac413f2482668c9f03ce4d07922"
```

Comment:
Why isn't the class hash the same as in `genesis-assets/genesis.json`?

Reply:
The ABI hash is used to compute the class hash. Madara use the ABI in `configs/genesis-assets/ERC20.json` and the Mock use `cairo-contracts/build/ERC20.json`. This files have the same content but the one used by the Mock have some more `\n`. You can check with `diff -u` Do you want i open an issue to uniform this ?

Reply:
Yeah you can open an issue and then close this conversation. 
Thanks a lot